README.md

```diff
@@ -65,13 +65,15 @@ Consequently, the API endpoints and options in the UI will be enabled/disabled.
 
 | Certificate type      | State |
 | ----------- | ----------- |
-| Vaccination      | enabled       |
-| Vaccination for tourists | disabled        |
-| Test (based on negative PCR or Rapid Antigen Test)   | enabled        |
-| Recovery (based on positive PCR test)   | enabled        |
-| Recovery (based on positive Rapid Antigen Test)   | disabled        |
-| Recovery (based on antibody test)   | disabled        |
-| Exceptional (based on a medical attestation of exemption)   | disabled        |
+| Vaccination      | :heavy_check_mark:       |
+| Vaccination for tourists | :no_entry:        |
+| Test (based on negative PCR or Rapid Antigen Test)   | :heavy_check_mark:        |
+| Recovery (based on positive PCR test)   | :heavy_check_mark:        |
+| Recovery (based on positive Rapid Antigen Test)   | :heavy_check_mark:        |
+| Recovery (based on antibody test)   | :no_entry:        |
+
```

Comment:
Weshalb ist Exceptional Zertifikat ganz aus der Liste entfernt?

Reply:
Gute Frage, die mir einige Ideen liefert, die wir morgen diskutieren könnten. Das Zertifikat des Typs "Exceptional" ist aus dem einfachen Grund nicht dokumentiert, weil dieses Zertifikat nur über die UI-Schnittstelle (also nicht über die API) erstellt werden kann.

Wenn ich jedoch einen Schritt zurücktrete, denke ich, dass hier ein einziger Einstiegspunkt für die Benutzerdokumentation fehlt. To be discussed